src/mmw/apps/modeling/mapshed/calcs.py

```diff
@@ -563,13 +563,17 @@ def normal_sys(lu_area):
     effectively dependent only on the area of medium density mixed land use.
     However, we replicate the original formula for consistency.
 
+    Returns an array of 12 ints as input for GWLF-E.
```

Comment:
Instead of saying "12 ints" (which describes the form) consider using something that describes the function, such as "Return a value for each month of the year."